merges/firefoxos/manifest.webapp

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "App name",
+  "description": "Some cool app for Firefox OS",
+  "launch_path": "/index.html",
+  "icons": {
+    "512": "/icon/icon-512.png",
+    "128": "/icon/icon-128.png",
+    "60": "/icon/icon-60.png"
+  },
+  "developer": {
+    "name": "Me",
+    "url": "http://me.com"
+  },
+  "default_locale": "en"
+}
```

Comment:
I have no idea why you need a separate format for FirefoxOS compared to iOS and Android, but I suspect it's because the Cordova tools for FirefoxOS just aren't there yet. :/